app/database/queries/breeze_buddy/chat_session.py

```
@@ -370,11 +377,47 @@ def insert_chat_message_query(
+        WHERE m.session_id = $1
+          AND m.idx > $2
+          AND COALESCE(m.sender_type, '') <> 'internal'
+          AND NOT (
+              m.idx = 0
+              AND EXISTS (
+                  SELECT 1 FROM {CHAT_SESSION_TABLE} cs
+                  WHERE cs.id = $1 AND cs.metadata ? 'human_assist_lineage'
+              )
+          )
+        ORDER BY m.idx ASC
+    """
```

Ruff (0.16.1)

[error] Possible SQL injection vector through string-based query construction

(S608)